flipperdevices/flipperzero-firmware
# [FL-3055] Getter for application data path

# What's new

- Storage: apps_data path getter, that ensures the path exists.

# Verification 

- Launch spi_mem_manager, music_player, picopass, hid_ble.
- Make sure the correct folder is created in /ext/apps_data and the app is using that folder.
- Make sure the data from the old application path is copied to the new path and the old path is deleted.

# Checklist (For Reviewer)
